src/InsertData.py

```diff
@@ -13,12 +13,17 @@
 from pymongo import MongoClient
 from pymongo import errors as mongoerrors
 
+QUARTER_INDEX = -16
```

Comment:
Should we refactor out this , since we use in 2 places already?

Reply:
I'm not sure what this comment means...

Reply:
"from pymongo import MongoClient, errors as mongoerrors" ?

Reply:
> "from pymongo import MongoClient, errors as mongoerrors" ?

Isn't the comment for ```QUARTER_INDEX = -16```